resources/application-connector/charts/application-broker/templates/deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
       - name: ctrl
         image: "{{ .Values.global.containerRegistry.path }}/application-broker:{{ .Values.global.application_broker.version }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+          - "--new-eventing-flow={{ .Values.global.isBEBEnabled }}"
```

Comment:
please use the flag as environment variable e.g.:

```yaml
- name: APP_NEW_EVENTING_FLOW
  value: {{ .Values.global.isBEBEnabled }}
```

and define it in [config file](https://github.com/kyma-project/kyma/blob/master/components/application-broker/internal/config/config.go#L29)

Reply:
Fixed